src/other/landingpage/story.js

```diff
@@ -12,16 +12,52 @@ story.addParameters({
   knobs: { disabled: true },
   changelog: { disabled: true },
   codepreview: { disabled: true },
+  a11y: { disabled: true },
 });
 
+const getFormattedGitCommitMessage = answerJson => {
+  const formattedMessage = answerJson.items[0].commit.message
+    .replace('Publish\n\n', '')
+    .replace(/- /g, '<br/>');
+  return formattedMessage;
+};
+
+const getDateFromGitCommit = answerJson => {
+  const formattedDate = new Date(answerJson.items[0].commit.author.date);
+  return formattedDate.toLocaleString(navigator.language);
+};
+
 story.add('to Pattern Library', () => {
   const wrapper = document.createElement('div');
 
+  const xhttp = new XMLHttpRequest();
```

Comment:
I suggest rewriting this with modern `fetch(...)` and async/await.

Reply:
I had fetch before. But we do not have a polyfill for that. And because it is just a story.js i choose this oldschool one ;)

Reply:
I see (though I don't feel our storybook presence needs to be fully cross-browser).